### DISCUSSION TITLE
How to be Automatically Verify Email id

### DISCUSSION THREAD

**Discussion by Krishna-ajmera-infotech** [chatwoot/chatwoot#4291] [Self hosted] [answered]
- How can set   Email verification features to be automatically verified to create a new Account by Chatwoot API?

**Comment by sojan-official** [comment, marked as answer]
@Krishna-ajmera-infotech  The users created using platform APIs are created as verified users. 
https://github.com/chatwoot/chatwoot/wiki/Building-on-Top-of-Chatwoot%3A-Platform-APIs
